modules/popup/react/lib/Popup.tsx

```diff
@@ -61,7 +61,7 @@ export interface PopupProps extends React.HTMLAttributes<HTMLDivElement> {
    * The `aria-label` for the Popup close button.
    * @default Close
    */
-  closeLabel?: string;
+  closeButtonAriaLabel?: string;
```

Comment:
note this is also used for the title

Reply:
Honestly I don't think we need a title for this icon. We generally don't apply titles to icon buttons. We could probably remove it, and it would make this clearer.

Reply:
I was still used for aria-label and I figured I'd keep it for translations